packages/mask/popups/pages/Wallet/WalletSettings/HidingScamTx.tsx

```diff
@@ -0,0 +1,32 @@
+import { Icons } from '@masknet/icons'
+import { Box, ListItem, Typography, useTheme, Switch } from '@mui/material'
+import { useMaskSharedTrans } from '../../../../../utils/index.js'
+import { useStyles } from './useStyles.js'
+import { hidingScamSettings } from '@masknet/shared-base'
+import { useValueRef } from '@masknet/shared-base-ui'
+
+export function HidingScamTx() {
+    const { t } = useMaskSharedTrans()
+    const theme = useTheme()
+    const { classes } = useStyles()
+    const hiding = useValueRef(hidingScamSettings)
+
+    return (
+        <ListItem className={classes.item}>
+            <Box className={classes.itemBox}>
+                <Icons.DangerOutline size={20} color={theme.palette.maskColor.second} />
```

Comment:
```
                <Icons.DangerOutline size={20} color={theme.palette.maskColor.second} />
                                               ^^^^^
```

This color won't take effect unless you set color in the svg file to `currentColor`. You can learn from https://mask.atlassian.net/wiki/spaces/MASK/pages/314441730